src/clincoded/static/components/provisional_classification/approval.js

```diff
@@ -298,11 +408,29 @@ const ClassificationApproval = module.exports.ClassificationApproval = createRea
                                     {affiliation && affiliation.length ?
                                         <div className="classification-approver">
                                             <dl className="inline-dl clearfix">
-                                                <dt><span>Approver:</span></dt>
+                                                <dt><span>Affiliation Approver:</span></dt>
                                                 <dd>{classificationApprover}</dd>
                                             </dl>
                                         </div>
                                         : null}
+                                    <div className="sop-version">
+                                        <dl className="inline-dl clearfix">
+                                            <dt><span>SOP Version:</span></dt>
+                                            <dd>{sopVersion}</dd>
+                                        </dl>
+                                    </div>
+                                    <div>
+                                        <dl className="inline-dl clearfix">
+                                            <dt><span>Curation Contributor(s):</span></dt>
+                                            <dd>{curationContributors ? renderContributorNames(curationContributors).join(', ') : null}</dd>
+                                        </dl>
+                                    </div>
+                                    <div>
+                                        <dl className="inline-dl clearfix">
+                                            <dt><span>Contributor Comments:</span></dt>
+                                            <dd><span>{contributorComment}</span></dd>
+                                        </dl>
+                                    </div>
                                 </div>
                                 <div className="col-xs-12 col-sm-3">
```

Comment:
I didn't notice before, but it looks like the approvers side of the panel is a bit squished when there are a lot of approvers. Not sure if there will typically be that many, but if so, I'd recommend changing `col-sm-3` to `col-sm-5` or `col-sm-6` since there is quite a bit of space, but up to you.
<img width="1234" alt="Screen Shot 2019-11-07 at 2 43 45 PM" src="https://user-images.githubusercontent.com/51174932/68435135-83a75280-016f-11ea-9298-1316f8d9e17a.png">